third_party/dependency_analyzer/src/test/io/bazel/rulesscala/dependencyanalyzer/AstUsedJarFinderTest.scala

```diff
@@ -452,4 +462,26 @@ class AstUsedJarFinderTest extends FunSuite {
       }
     }
   }
+
+  test("classOf in class Java annotation is direct") {
```

Comment:
What is this test?

Reply:
It makes sure that if you use a `classOf[X]` in a Java annotation, that `X` is considered a direct dependency. Scalac handles java and scala annotations differently hence the need for an explicit java test.

Should we change this to just use A/B instead of Category/UnitTests?

Reply:
It's ok I just didn't understand why the test was added and how it relates to locations in error report.
Also,
It's for coverage, right? You didn't also add a fix here AFAICT